src/LibYear.Core/ProjectFileManager.cs

```diff
@@ -10,10 +10,10 @@ public class ProjectFileManager : IProjectFileManager
 	public ProjectFileManager(IFileSystem fileSystem)
 		=> _fileSystem = fileSystem;
 
-	public async Task<IReadOnlyCollection<IProjectFile>> GetAllProjects(IReadOnlyCollection<string> paths)
+	public async Task<IReadOnlyCollection<IProjectFile>> GetAllProjects(IReadOnlyCollection<string> paths, bool recursive = false)
 	{
 		if (!paths.Any())
-			return await GetProjectsInDir(Directory.GetCurrentDirectory());
+			return await GetProjectsInDir(Directory.GetCurrentDirectory(), recursive);
 
 		var tasks = paths.Select(GetProjects);
```

Comment:
I think we need to pass the `recursive` arg here as well, in case the user sends multiple directories that they want to search